### ISSUE TITLE
[trigger-sim] I3GlobalTriggerSim and I3EventHeader (Trac #1388)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1380]
We need an I3EventHeader for untriggered events too.  Write a test and maybe some docs to make sure we don't accidentally remove this years from now.

Migrated from https://code.icecube.wisc.edu/ticket/1388
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "We need an I3EventHeader for untriggered events too.  Write a test and maybe some docs to make sure we don't accidentally remove this years from now.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "combo simulation", 
    "summary": "[trigger-sim] I3GlobalTriggerSim and I3EventHeader", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-10-07T14:55:56", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151104T19:49:26: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160119T04:04:00: 
* olivas changed status from "accepted" to "closed"
* olivas commented:

> In changeset 141005:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141005"
> cleaned up test.  added an explicit check for I3EventHeader for untriggered events.  closes #1380.
> ```


* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas changed milestone from "IceSim 5.0" to ""
* olivas commented:

> Milestone IceSim 5.0 deleted
